src/xo-app/vm/tab-network.js

```diff
@@ -33,35 +35,52 @@ import {
   connectVif,
   createVmInterface,
   deleteVif,
+  deleteVifs,
   disconnectVif,
   isVmRunning,
   setVif,
   subscribeIpPools,
   subscribeResourceSets,
 } from 'xo'
 
-const IP_COLUMN_STYLE = { maxWidth: '20em' }
-const TABLE_STYLE = { minWidth: '0' }
+@connectStore({
+  vif: (_, props) => props.item,
```

Comment:
You don't need this one since the VIF is already passed through `this.props.item`.